pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -26,6 +27,7 @@
 , disableBootstrap ? stdenv.targetPlatform != stdenv.hostPlatform
 }:
 
+assert disableGdbPlugin -> enablePlugin;
```

Comment:
Worth removing added assert? It should be fine to disable both.

Reply:
Dammit, that should be `!enablePlugin`.  Thank you for noticing.

Reply:
Dammit, that should be `!enablePlugin`.  Thank you for noticing.